packages/regl-worldview/src/commands/Command.js

```diff
@@ -133,13 +137,20 @@ function getHitmapProps(getHitmapId, children) {
   }
 
   return children.reduce((memo, marker) => {
-    const hitmapId = getHitmapId(marker);
-    // filter out components that don't have hitmapIds
-    if (hitmapId != null) {
+    if (marker.colors) {
       memo.push({
         ...marker,
-        color: intToRGB(getHitmapId(marker) || 0),
+        colors: marker.colors.map((_, colorIndex) => intToRGB(getHitmapId(marker, colorIndex) || 0)),
```

Comment:
Unless I'm reading this wrong, it seems like the hitmap ids will be different depending on whether the input marker had `color` or `colors`. That doesn't really make sense to me; the hitmap id is a very separate feature from the visible color.

Reply:
Good catch. Updated to use `points` field instead